README.md

```diff
@@ -28,16 +28,17 @@ changes may be written to a config file in a platform-specific location:
 
 ## Build and installation
 
-- **Install Go 1.19 or 1.20**
+- **Install Go 1.19 or higher version**
```

Comment:
Not sure why this mentions 1.20 as "upper bound" there, it seems to be working fine with 1.21 for me on Linux and since minor Golang versions **should** be backward-compatible,

besides, if I have 1.21 installed I probably don't want to install/switch to 1.20 to install `dcrctl` specifically.

Reply:
It just hadn't been updated it for Go 1.21 yet.  We only officially support the latest version of Go and the one prior, hence the explicit versions.  It's fine to say or higher though since it's really the lower bound that matters most.

In practice, it should work back to go1.17 for now since both its `go.mod` and those of its dependencies only require that version.  However, we don't officially support things that far back and I would imagine some of the deps will be bumping to a newer version to be able to use newer features before too much longer.